rules/internal/objc_provider_utils.bzl

```diff
@@ -57,6 +57,8 @@ def _merge_dynamic_framework_providers(dynamic_framework_providers):
 
 def _merge_cc_info_providers(cc_info_providers, merge_keys = [
     "headers",
+    "includes",
+    "defines",
```

Comment:
Just adding `defines` here to be consistent with `apple_framework_packaging`, not 100% sure if we absolutely need it